util/csvresponse.py

```diff
@@ -14,6 +14,13 @@
 PRESSURE_DPI = app.config.get('PRESSURE_DPI')
 INT_PRESSURE_NAME = app.config.get('INT_PRESSURE_NAME')
 
+# constants for filtering QC Parameters
+QC_EXECUTED = 'qc_executed'
```

Comment:
These `QC_EXECUTED, QC_RESULTS` would be better placed in `qc_executor.py` and imported into here, `netcdf_generator.py` and `jsonresponse.py`.

Reply:
Good point, will implement.

Reply:
qc_executor.py uses literals of "_qc_executed" and "_qc_results" around line 140. For 12189 I need them as well in stream_request.py. Is there any reason in this file you need to exclude the leading underscore, now that you're using the split function?

Reply:
The underscore is a joining character, so it shouldn't be included. When you need it, you can add it in a pythonic way (note join takes a list):

```qc_name = '_'.join([param_name, QC_EXECUTED])```

Reply:
Makes sense, so alter those lines in qc_executor.py accordingly while adding those variables to the file.